package.json

```diff
@@ -11,19 +11,18 @@
   },
   "homepage": "https://github.com/Universal-Connect-Project/ucp-hosted-app#readme",
   "scripts": {
-    "build": "turbo build",
+    "build": "",
```

Comment:
Why remove this?

Reply:
I thought this was breaking the build process but just tested adding it back and it doesn't break so I'll leave it.